src/Red.jsx

```diff
@@ -0,0 +1,37 @@
+import { React, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+
+export default function Red(props) {
+  const { AllshoesName, AllImage } = props;
```

Comment:
it would be simpler if this component just took a single prop called `shoes` that had both the information and the images; try changing this component and its parent to do it with a single prop